package.json

```diff
@@ -22,7 +22,9 @@
     "lint": "prettier --check '**/*.*' && eslint .",
     "prepublishOnly": "yarn build",
     "semantic-release": "semantic-release",
-    "test": "jest --coverage && yarn lint"
+    "test": "jest --coverage && yarn lint",
+    "generate-ts-declarations": "tsc"
```

Comment:
Can this be done at build time too? So that it automatically updates on publish. Or at least have a way so that in tests it would fail if TypeScript definition is not up to date.